Source/diablo.cpp

```diff
@@ -289,8 +289,15 @@ void diablo_splash()
 	UiTitleDialog();
 }
 
+void diablo_terminate(){
+    init_cleanup();
+    SaveGamma();
+    SDL_Quit();
+}
+
 int APIENTRY WinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPSTR lpCmdLine, int nCmdShow)
 {
+    atexit(diablo_terminate);

```

Comment:
Indentation on this line is wrong

Reply:
What is the correct spacing and indentation?